### DISCUSSION TITLE
[OKD-SCOS v4.16] Unable to provision SNO cluster on baremetal using Assisted Installer service

### DISCUSSION THREAD

**Discussion by akiselev1** [okd-project/okd#2002] [Pre-Release Testing]
I tried the latest 4.16 image, available in Quay: 4.16.0-0.okd-scos-2024-08-01-132038. This installation was driven by Assisted Installer service running in Sylva management cluster. 

I saw Ironic inspection passed,  the node rebooted from virtual CD, but then installation hung indefinitely at "Host: okd-sno, reached installation stage Waiting for bootkube"

 Further investigation showed that bootkube service running on the BM server was stuck in a permanent fail/restart loop due to the missing oc binary:

```
Aug 13 21:13:04 okd-sno bootkube.sh[291892]: /usr/local/bin/bootkube.sh: line 81: oc: command not found
Aug 13 21:13:04 okd-sno systemd[1]: bootkube.service: Main process exited, code=exited, status=127/n/a
Aug 13 21:13:04 okd-sno systemd[1]: bootkube.service: Failed with result 'exit-code'.
```

**Comment by titou10titou10** [comment]
maybe you could check my post here 
https://github.com/okd-project/okd/discussions/2015

not exactly the same error but also with an ABI UPI install

**Reply by akiselev1** [reply]
Thank you for the pointer! That may the same issue.  I didn't have a chance to dig deeper beyond the error above. Read-only /sysroot would do it. I will try to use your workaround with rhcos OS image and see if that helps in my case.

**Reply by akiselev1** [reply]
I confirmed that this is the same issue as #2015 with 4.16.0-0.okd-scos-2024-09-27-110344 image. RHCOS workaround did help.

**Comment by akiselev1** [comment]
This
